### ISSUE TITLE
Is it possible to add Russian language support?

### ISSUE THREAD

**Issue by Andrew666** [Ephemerality/xray-builder.gui#62]

**Comment by Ephemerality** [comment]
Like a translation for the UI or is there an issue with Cyrillic-encoded books?

**Comment by Andrew666** [comment]
There an issue with Cyrillic-encoded books

**Comment by Ephemerality** [comment]
What's the issue and do you have an example book you could share?

**Comment by Andrew666** [comment]
Since the book is in Cyrillic, the program does not find comparisons to English terms. I have attached the log and the book file.
[19.08.13.12.05.2024.txt](https://github.com/Ephemerality/xray-builder.gui/files/15286976/19.08.13.12.05.2024.txt)
[Ighry Niemiezidy - Dzhieims Kori.zip](https://github.com/Ephemerality/xray-builder.gui/files/15286981/Ighry.Niemiezidy.-.Dzhieims.Kori.zip)

**Comment by Ephemerality** [comment]
Sorry for the delay -- do you have a source somewhere that lists the terms in the Cyrillic form that show up in the book? My current term source doesn't have any Russian translations available.

**Comment by Andrew666** [comment]
Нет, к сожалению я знаю только русскую википедию. Я думал может получится к русскоязычным книгам добавлять аналогичные термины из английских источников. В любом случае спасибо за реакцию.

**Comment by Andrew666** [comment]
No, unfortunately I only know Russian Wikipedia. I thought it might be possible to add similar terms from English sources to Russian-language books. In any case, thanks for the reaction.

**Comment by Ephemerality** [comment]
Yeah the issue is that they have to match exactly with the text of the book
so if the names are in Cyrillic the book I'd need to have a source for
those exact strings. For example, the app supports German because the data
source has German character names available.
